src/simularium/ISimulator.ts

```diff
@@ -0,0 +1,68 @@
+//import jsLogger from "js-logger";
+//import { ILogger } from "js-logger";
+
+import { VisDataMessage, TrajectoryFileInfoV2 } from "./types";
+
+// TODO we need to examine this whole interface and document it with comments here
+// what is each function expected to do, from the caller's perspective?
+// are they named appropriately?
+// and are some of them completely specific to websockets only?
+export interface ISimulator {
```

Comment:
I think this interface should have some sort of a doc block

Reply:
I want to clean up this whole interface in another PR